src/pages/MonthlyGood/HowItWorks.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	<div class="how-works-section-wrapper">
```

Comment:
This wrapping div is unnecessary and the `.row` div can be used as the root element instead.

Reply:
This one is necessary because of the full width background color

Reply:
You're right, sorry, I missed that one. Confused it with `.learn-more-wrapper`